From 3cf212c4ce6cd72c09bc47f35f539ba0afd4d106 Mon Sep 17 00:00:00 2001
Message-Id: <3cf212c4ce6cd72c09bc47f35f539ba0afd4d106.1678716918.git.lorenzo@kernel.org>
From: Lorenzo Bianconi <lorenzo@kernel.org>
Date: Sun, 12 Mar 2023 16:40:31 +0100
Subject: [PATCH net-next 1/2] net: ethernet: mtk_wed: rename
 mtk_wed_get_memory_region in mtk_wed_get_reserved_memory_region

This is a preliminary patch to move wed ilm/dlm and cpuboot properties in
dedicated dts nodes.

Signed-off-by: Lorenzo Bianconi <lorenzo@kernel.org>
---
 drivers/net/ethernet/mediatek/mtk_wed_mcu.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/net/ethernet/mediatek/mtk_wed_mcu.c
+++ b/drivers/net/ethernet/mediatek/mtk_wed_mcu.c
@@ -234,8 +234,8 @@ int mtk_wed_mcu_msg_update(struct mtk_we
 }
 
 static int
-mtk_wed_get_memory_region(struct mtk_wed_hw *hw, int index,
-			  struct mtk_wed_wo_memory_region *region)
+mtk_wed_get_reserved_memory_region(struct mtk_wed_hw *hw, int index,
+				   struct mtk_wed_wo_memory_region *region)
 {
 	struct reserved_mem *rmem;
 	struct device_node *np;
@@ -321,7 +321,7 @@ mtk_wed_mcu_load_firmware(struct mtk_wed
 		if (index < 0)
 			continue;
 
-		ret = mtk_wed_get_memory_region(wo->hw, index, &mem_region[i]);
+		ret = mtk_wed_get_reserved_memory_region(wo->hw, index, &mem_region[i]);
 		if (ret)
 			return ret;
 	}
